issues.md

```diff
@@ -0,0 +1,26 @@
+# Issue reporting guidelines
+
+## General guidelines
+
+An issue must be an actionable task.
+It must be clear at a glance what needs to be done for an issue to be resolved.
+
+**Issue titles:**
+
+- The title of an issue has to convey succinctly what the task is.
+
+**Description:**
+An issue's description should list what needs to be done and the status of the task or sub-tasks.
+It should be clear to a reader who has no context on the issue what the current status of the task is and what is left outstanding.
+The scope and status of an issue can be conveyed via a checklist or using labels.
+
+- It should not be necessary to scroll through comments or replies to find the status of the task or sub-tasks.
+- The status of the task (or sub-tasks) must be visible in the issue description or via labels.
+- Updates to the status of the task should be done by editing the description, not by adding a comment reply to an issue.
```

Comment:
So this is fine in theory, but with FF issues specifically most assignees add their own checklist to the issue to compliment the original one (like a list of files which need amending)

Maybe 'updates should be done by editing task lists or description'?

Reply:
great, have added details about task list in the description